### PR TITLE
Fix git-lfs track command in Performance

### PR DESCRIPTION
**Pull request by albertvillanova**
There is a mistake in the `git-lfs track` command in the Performance: images are not properly tracked by Git LFS, thus resulting in a git remote error while pushing.

This PR fixes the `git-lfs track` command, so that all images are properly tracked by Git LFS.

### CODE REVIEW

File: Performance.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Executed in   38.87 mins
 Compare this to a system like [git lfs](https://git-lfs.github.com/) on the same dataset
 
 ```
-git lfs track images # ~17 sec
+git lfs track *.jpg
```
Comment (julien-c) [inline Performance.md:45]:
```suggestion
git lfs track "*.jpg"
```

Comment (albertvillanova) [reply Performance.md:45]: Thanks @julien-c

**Comment by gschoeni** [comment]
@albertvillanova we were wondering if something was off, thank you! Let me try again and update the performance numbers.

**Comment by gschoeni** [comment]
Verfied this works with `git lfs ls-files` 👍

**Comment by albertvillanova** [comment]
Great!

@gschoeni please note that the fix works because all the files inside the "images" have ".jpg" extension. Alternatively, to track all the files inside the "images" directory, independently of their extension, you could have used:
```
git lfs track "images/*"
```
